### PR TITLE
Docsite: add Community guide

### PR DESCRIPTION
**Pull request by Andersson007**
Dear maintainers,
As a part of the [Consolidating Ansible discussion platforms initiative](https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812), this PR adds the communication section template defined by the community to the README and the docsite. Similar PRs are now being raised across all the Ansible ecosystem projects.

Thank you

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @oraNod

File: docs/source/index.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@ Ansible SDK documentation
 
 Ansible SDK is a toolkit that lets you harness the power and simplicity of Ansible automation directly from your applications.
 
+.. note::
+
+  Need help or want to discuss Ansible SDK including the documentation? See the :ref:`Community guide<community>` to learn how to join the conversation!
```
Comment (oraNod) [inline docs/source/index.rst:8]:
```suggestion
  Need help or want to discuss all things related to Ansible SDK? See the :ref:`Community guide<community>` and join the conversation!
```

File: docs/source/community.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+* `Get Help <https://forum.ansible.com/c/help/6>`_: get help or help others. Please add appropriate tags if you start new discussions, for example `ansible-sdk`.
+* `Posts tagged with 'ansible-sdk' <https://forum.ansible.com/tag/ansible-sdk>`_: subscribe to participate in project-related conversations.
+* `Bullhorn newsletter <https://docs.ansible.com/ansible/devel/community/communication.html#the-bullhorn>`_: used to announce releases and important changes.
+* `Social Spaces <https://forum.ansible.com/c/chat/4>`_: gather and interact with fellow enthusiasts.
+* `News & Announcements <https://forum.ansible.com/c/news/5>`_: track project-wide announcements including social events.
+
+For more information on the forum navigation, see `Navigating the Ansible forum <https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39>`_ post.
```
Comment (oraNod) [inline docs/source/community.rst:25]:
```suggestion
See `Navigating the Ansible forum <https://forum.ansible.com/t/navigating-the-ansible-forum-tags-categories-and-concepts/39>`_ for some practical advice on finding your way around.
```

**Comment by Andersson007** [comment]
@oraNod thanks for reviewing! All committed, PTAL

File: docs/source/index.rst [hunk 3]
```diff
@@ -3,6 +3,10 @@ Ansible SDK documentation
 
 Ansible SDK is a toolkit that lets you harness the power and simplicity of Ansible automation directly from your applications.
 
+.. note::
+
+  Need help or want to discuss all things related to Ansible SDK? See the :ref:`Community guide<community>` and join the conversation!
```
Comment (oraNod) [inline docs/source/index.rst:8]:
```suggestion
   Need help or want to discuss all things related to Ansible SDK? See the :ref:`Community guide<community>` and join the conversation!
```

**Review by oraNod — APPROVED** [review]
LGTM

**Pull request merged**